marconi-sidechain-node/marconi-sidechain-node/marconi-sidechain-node.cabal

```diff
@@ -52,11 +52,11 @@ library
   --------------------------
   build-depends:
     , cardano-crypto-class
-    , cardano-node               ==8.1.2
+    , cardano-node               ==8.4.0
```

Comment:
Did you check if the `main` function of  `cardano-node` change between these versions so that you copy it in the `Run` module.

Reply:
I forgot to do that. I'll check it after the standup.